src/test/scala/ironoxide/FullIntegrationTest.scala

```diff
@@ -151,16 +142,14 @@ class FullIntegrationTest extends AsyncWordSpec with Matchers with EitherValues
           .unsafeRunSync
           .value
 
-      // what was valid should go through
-      (result.changed.getUsers should have).length(1)
-      result.changed.getUsers.head.getId shouldBe primaryTestUserId.id
-      (result.changed.getGroups should have).length(0)
-
-      // the invalid stuff should have errored
-      (result.errors.getUsers should have).length(0)
-      (result.errors.getGroups should have).length(1)
-      result.errors.getGroups.head.getId.getId shouldBe s"data_recovery_${primaryTestUserId.id}"
-      result.errors.getGroups.head.getErr should include("Policy refers to unknown user or group")
+      result.changed shouldBe List(primaryTestUserId)
+      val missingGroupId = s"data_recovery_${primaryTestUserId.id}"
+      result.errors should contain theSameElementsAs List(
```

Comment:
Is there a reason you wouldn't just check equality since they're both of length one?